index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+
+  <nav id="navbar">
+  <header>User's Manual</header>
+ 
+    
+    <ul>
+      <li><a class="nav-link" href="#Setting_up_your_oven"> Setting up your oven </a></li>
+      <li><a class="nav-link" href="#Clock_Setting"> Clock Setting </a></li>
+      <li><a class="nav-link" href="#Operation"> Operation </a></li>
+      <li><a class="nav-link" href="#Maintenance"> Maintenance </a></li>
+      <li><a class="nav-link" href="#Warranty_information"> Warranty information </a></li>
+       </nav>
+
+<head>
+  <body>
+  <h1>Microwave user's manual for Dummies</h2>
```

Comment:
Nice job pointing this out :+1: However, It would be better if you could make suggestions instead of giving an order to sound nicer. (I'm referring to the first phrase `Be careful ...` ).